datadotworld/client/api.py

```diff
@@ -1578,20 +1516,11 @@ def __build_dataset_obj(dataset_constructor, file_constructor, args):
         if 'visibility' in args:
             dataset.visibility = args['visibility']
 
-        dataset.files = files
-
         return dataset
 
     @staticmethod
-    def __build_project_obj(project_constructor, file_constructor, args):
+    def __build_project_obj(project_constructor, args):
```

Comment:
Did we disable the files property on projects too? Or was it exclusively datasets.

Reply:
The files property was removed from `update_project`, but URLs are still enabled for `create_project` and `replace_project`. Will update